### PR TITLE
Backmerge v2.7.2-rc into master

### PR DESCRIPTION
**Pull request by anthonysena**
Merge in updates for 2.7.2 release

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,15 +6,14 @@
   <groupId>org.ohdsi</groupId>
   <artifactId>WebAPI</artifactId>
   <packaging>war</packaging>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>2.7.2</version>
```
Comment (pavgra) [inline pom.xml]: Why not `2.7.3-SNAPSHOT`?

Comment (chrisknoll) [reply pom.xml]: if this is going on master, it should probably be 2.8.0-SNAPSHOT. If we need to do a point release, we'll branch off the 2.7.2 release tag, and set the POM version accordingly.

Comment (anthonysena) [reply pom.xml]: Good catch here @pavgra. I agree with @chrisknoll here - putting 2.8.0-SNAPSHOT on master at this point makes sense. Should we need a 2.7.3, we'll branch from the 2.7.2 release tag. I've updated the PR accordingly. Thanks!

**Comment by pavgra** [comment]
@anthonysena , seems like you've updated it only in one place while 2 are required

image

**Comment by anthonysena** [comment]
Sorry for that - I've adjusted this so that the `webapi.version` is 2.8.0 while the `version` is 2.8.0-SNAPSHOT. The former is displayed through the /info endpoint and thought it would be confusing if the -SNAPSHOT reference was there. 

If you do the merge, please squash down these repetitive commits. Thanks

**Pull request merged**